### ISSUE TITLE
Astaire's init.d script doesn't set namespace_prefix so it doesn't work when using multiple networks

### ISSUE THREAD

**Issue by dtheobald** [Metaswitch/astaire#19]
Astaire's init.d script looks like it is missing the setting of 'namespace_prefix' in get_settings, so I added it to mine:

```
get_settings()
{
        # Set up defaults and then pull in the settings for this node.
        . /etc/clearwater/config

        # Set up defaults for user settings then pull in any overrides.
        log_level=2
        [ -r /etc/clearwater/user_settings ] && . /etc/clearwater/user_settings
        [ -z "$signaling_namespace" ] || namespace_prefix="ip netns exec $signaling_namespace"
}
```

**Comment by bossmc** [comment]
Good spot, can you raise a PR with this fix please?

**Comment by dtheobald** [comment]
Will do.

**Comment by JimBunch** [comment]
This issue is high priority because we have customers using the multiple networks feature.

**Comment by rkd-msw** [comment]
Fixed by #20.
